recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in

```diff
@@ -1,4 +1,4 @@
 fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
-fatload mmc 0:1 ${kernel_addr_r} @@KERNEL_IMAGETYPE@@
-if test ! -e mmc 0:1 uboot.env; then saveenv; fi;
+fatload @@BOOT_DEVTYPE@@ 0:1 ${kernel_addr_r} @@KERNEL_IMAGETYPE@@
```

Comment:
UBOOT_ROOT_DEVTYPE would be my pick here. What do you think?

Reply:
Call it BOOT_MEDIA :)